sys/include/net/someip.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2020 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_someip SOME/IP
+ * @ingroup     net
+ * @brief       Provides SOME/IP header and field defines
+ *
+ * @{
+ *
+ * @file
+ * @brief   SOME/IP header and field definitions
+ *
+ * @author  Jannes Volkens <jannes.volkens@haw-hamburg.de>
+ * @}
+ */
+#ifndef NET_SOMEIP_H
+#define NET_SOMEIP_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <stdint.h>
+
+#define SOMEIP_MAX_PLEN (1400)    /**< SOME/IP maximum payload length */
+#define SOMEIP_HDR_LENGTH (8)     /**< SOME/IP header length size, excluding payload */
+#define SOMEIP_FULL_HDR_SIZE (16) /**< SOME/IP full header size, including all fields except for the payload */
+
+/**
+ * @brief       Structure of the Message ID
+ */
+    struct message_id
+    {
+        uint16_t service_id; /**< Service ID */
+        uint16_t method_id;  /**< Method ID */
+    };
```

Comment:
Indentation, style, and namespacing. Also we typically use `typedef`s even for structs

```suggestion
typedef struct {
    uint16_t service_id; /**< Service ID */
    uint16_t method_id;  /**< Method ID */
} someip_message_id_t;
```